docker/activemq/README.md

```diff
@@ -1,18 +1,76 @@
 ActiveMQ-docker
 ============
-Dockerfile source for ActiveMQ [docker](https://docker.io) image.
+Dockerfile source for ActiveMQ [docker](https://docker.io) image..
 
-# Quick Start for ActiveMQ 5.x
+## Upstream
+This source repo was originally copied from:
 
-You can launch the image using the `docker` command line:
+
+## Disclaimer
+This is not an official Google product.
+
+## About
+This image contains an installation of ActiveMQ
+
+For more information, see the
+[Official Image Marketplace Page](https://console.cloud.google.com/marketplace/product/bitnami-launchpad/activemq).
+
+### Prerequisites
+
+Configure [gcloud](https://cloud.google.com/sdk/gcloud/) as a Docker credential helper:
+
+```shell
+gcloud auth configure-docker
+```
+### Pull command
+
+```shell
+docker -- pull marketplace.gcr.io/google/activemq5
+```
+## Table of Contents
+
+ [Using Docker](#using-docker)
```

Comment:
No link in Table of Contents is taking the user to the right place

Reply:
added links